MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,12 +1,18 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
```

Comment:
Please, add `[MaxLength]` validation for this field. Max length should be somewhat around 100, maybe even a little bit less restricting.